src/style.scss

```diff
@@ -102,20 +102,38 @@
 
 .button--left {
   left: 0;
+  z-index: +1; // Ensure the gradient is behind the button without changing the order of the DOM elements
```

Comment:
Never seen this before. I was trying to look it up. The `+` does not seem to serve any purpose.

Reply:
+1 just means to treat it as high level higher in z-index than what it is by default. In this case, it bring's it above the gradient that is after it in the DOM.

Pull this down, delete the z-index rule, run the demo, scroll a carousel to the right (click on the right button) and you will see the issue on the left button being behind the gradient.